lib/rucio/daemons/bb8/bb8.py

```diff
@@ -282,7 +283,7 @@ def run_once(
 
                     logger(
                         logging.INFO,
-                        "Rebalance %d TB from %s(%f) to %s(%f)%s",
+                        "Rebalance %f TB from %s(%f) to %s(%f)%s",
```

Comment:
I think you will want `%.3f` or something here so that you don't have a lot of decimal places.